cra-rxjs-styled-components/src/routes/repo/index.tsx

```diff
@@ -0,0 +1,23 @@
+import { Outlet } from 'react-router';
+import { RepoLayout } from '../../components/layouts/RepoLayoutPage';
+import Header from '../../components/header';
+import SubHeader from '../../components/sub-header';
+import RepoPage from '../../components/repo-page';
+import { useParams } from 'react-router-dom';
+
+function Repo() {
+  const { username, repo, branch } = useParams();
+  return (
+    <>
+      <Header />
+      <RepoPage name={repo!} owner={username!} branch={branch!}>
```

Comment:
I don't really like this non null assertion. `!`

Reply:
Sorry about this working on alternative with ts